assets/locales/cs.json

```diff
@@ -1,25 +1,25 @@
 {
   "COMMON": {
     "Login": "Přihlásit se",
-    "Logout": "Odhlásit se",
-    "Sort": "Třídit",
+    "Logout": "Odhlásit",
+    "Sort": "Seřadit",
     "abstract": "Abstrakt",
-    "addAs": "Add as",
+    "addAs": "Přidat jako",
     "advancedOptions": "Pokročilé možnosti",
-    "all": "All",
-    "application": "Application",
+    "all": "Vše",
+    "application": "Aplikace",
     "area": "Plocha",
     "attribute": "Atribut",
     "attributes": "Atributy",
-    "boundingBox": "Ohraničující rámeček",
+    "boundingBox": "Ohraničující obdélník",
```

Comment:
Just an idea... "Rozsah", "Rozsah v mapě" could be better?

Reply:
FTR, "Ohraničující obdélník" is an official translation of bounding box, used in professional literature. But yes, for common users "Rozsah mapy" might be better understood. 👍

Reply:
Exactly.